src/Symfony/Bridge/Doctrine/PropertyInfo/DoctrineExtractor.php

```diff
@@ -50,7 +50,17 @@ public function getProperties($class, array $context = array())
             return;
         }
 
-        return array_merge($metadata->getFieldNames(), $metadata->getAssociationNames());
+        $properties = array_merge($metadata->getFieldNames(), $metadata->getAssociationNames());
+
+        if (class_exists('Doctrine\ORM\Mapping\Embedded') && $metadata instanceof ClassMetadataInfo && $metadata->embeddedClasses) {
```

Comment:
@dunglas, when class `Doctrine\ORM\Mapping\Embedded` doesn't exist, `$metadata->embeddedClasses` is not defined. I can only move `instanceof` check to the first place.